01.fizzbuzz/FizzBuzz.rb

```diff
@@ -1,21 +1,11 @@
-
-
-(1..20).each do |i|
-  case i % 15
-  when 0
+(1..20).each do |x|
```

Comment:
あ、前回のコメントでは変数xを使ったif文を書きましたが、あれはあくまでサンプルコードなので、変数名まで真似しなくていいです。
数値の場合はnumberのnやintegerのiを変数名にすることが多いですね。

```suggestion
(1..20).each do |n|
```

あわせて読みたい
https://blog.jnito.com/entry/2020/10/20/092724

Reply:
コメントありがとうございます。
記事読みました。
命名って何でもいいや〜って思ってました😨
改めます🙇‍♀️